### PR TITLE
Add global concurrency to the concurrency page

### PR DESCRIPTION
**Pull request by zhen0**
Adds a global concurrency tab and task run concurrency to one concurrency page. 

image

Wait on https://github.com/PrefectHQ/prefect-ui-library/pull/1719 and ui-library update before merging

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [ ] This pull request includes tests or only affects documentation.
- [ ] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [ ] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed

### CODE REVIEW

**Comment by zhen0** [comment]
Waiting on a ui library update but would welcome a review of the code here!

File: ui/src/pages/ConcurrencyLimits.vue [hunk 1]
```diff
@@ -1,19 +1,55 @@
...
+  
+  const { showModal: showModalGlobal, open: openGlobal } = useShowModal()
+  const { showModal: showModalTaskRun, open: openTaskRun } = useShowModal()
+
+  const computedTabs = computed(() => [
+    { label: 'Global' },
+    { label: 'Task Run' },
+  ])
+
+  const { tab, tabs } = useTabs(computedTabs)
```
Comment (pleek91) [inline ui/src/pages/ConcurrencyLimits.vue:41]: Can you create a tab ref using `useRouteQueryParam`? I update all the other pages with tabs to do that.

Comment (pleek91) [inline ui/src/pages/ConcurrencyLimits.vue:41]: Also you don't actually need to use this composition unless you have dynamic tabs.

File: ui/src/pages/ConcurrencyLimits.vue [hunk 2]
```diff
@@ -1,19 +1,55 @@
...
 
   <script lang="ts" setup>
-  import { PageHeading, ConcurrencyLimitsCreateModal, ConcurrencyLimitsTable, useShowModal } from '@prefecthq/prefect-ui-library'
+  import { PageHeading, ConcurrencyLimitsV2Table, ConcurrencyLimitsTable, ConcurrencyLimitsCreateModal, ConcurrencyLimitsV2CreateModal, useShowModal, useTabs } from '@prefecthq/prefect-ui-library'
+  import { computed } from 'vue'
+  
+  const { showModal: showModalGlobal, open: openGlobal } = useShowModal()
+  const { showModal: showModalTaskRun, open: openTaskRun } = useShowModal()
+
+  const computedTabs = computed(() => [
```
Comment (pleek91) [inline ui/src/pages/ConcurrencyLimits.vue:36]: This doesn't need to be computed.

**Review by pleek91 — APPROVED** [review]

**Pull request merged**